### ISSUE TITLE
[Accessibility] [MAS36-Keyboard] The close button can not be tabbed in persona picker item 

### ISSUE THREAD

**Issue by luvankle** [OfficeDev/office-ui-fabric-react#3552]
### Bug Report

Env: SCU
Browse: IE/Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Yes

Requested priority: High

Products/sites affected: Office.com share control

#### Describe the issue:

Repro steps:
1. Log in 
https://scuprodprv.office.com/?auth=2&flight=mrutiles,sharesapi,DocumentActivity,createvisio,BatchShares,TrendingDocs,NewActionsMenuDesign,NewMruActions,AppGallery,MruColumns,mrushare,ShowOrganizerImages,docpermissions,RectangularTiles,FluentLayout,commondate,places,FastSearchUX,RecommendedView,LifecycleEvents,AppGalleryFluent,SharePoint,fluentlayoutbackground
 with valid account.
2. Share a recent document.
3. Please see attachment.

#### Actual behavior:

User is able to tab to delete persona button

#### Expected behavior:

User is unable to tab to delete persona button

### If applicable, please provide a codepen repro:
Bug located here: https://office.visualstudio.com/OC/_workitems?id=2028744&_a=edit
Can contact luvankle@microsoft.com for questions

![image](https://user-images.githubusercontent.com/34457541/33850991-786e8eee-de6a-11e7-9cfb-c0a9bf2680be.png)

**Comment by betrue-final-final** [comment]
Does this still repro?

**Comment by luvankle** [comment]
Hi, yes this still repros.

A user is unable to tab to the delete button in the Share Control on Office.com, as shown in the photo.
________________________________

**Comment by aditima** [comment]
This is a bug we should fix. @joschect - could you please help?

**Comment by jordandrako** [comment]
To add some detail, each persona in the picker can be focused and deleted with the backspace key, but since the close button is a button, it should be focusable.
